### PR TITLE
nicer output

### PR DESCRIPTION
**Pull request by Xalava**
Modification to count and display the number of media extracted. 
( NB, should be inside the `else` of the precedent PR)

### CODE REVIEW

File: extract-media [hunk 1]
```diff
@@ -12,10 +12,14 @@ _extract() {
...
-  echo "$MEDIA_FILES" | while read -r line; do
+  MEDIA_NB=0
+  
+  # echo "$" |
+  while read -r line; do
+    MEDIA_NB=$((MEDIA_NB+1))
     cp "$line" "$2/"
-  done
+  done <<< "$MEDIA_FILES"
+  echo "$MEDIA_NB media succesfully extracted"
```
Comment (stefanjudis) [inline extract-media]: I wonder if there should be a verbose flag or something. Unix tools by default are very quiet. :) Thoughts?

**Comment by stefanjudis** [comment]
@Xalava Thanks so much. There are now merge conflicts and I (hopefully) set up Circle CI to run properly. Do you have time to have a quick look?

File: extract-media (hunk 1 above)
Comment (Xalava) [reply extract-media]: This is a good question. Surveying : 
- tar, cp, rm have no output by default, and a verbose option
- wget, zip, git have pretty verbose output and a quiet option. 

Intuitively, modern CLI tend to have some output, probably as they are higher level than base unix commands. 

So I suggest a one liner to give user feedback, and potentially a verbose option with each file and a quiet option

Comment (stefanjudis) [reply extract-media]: Fair enough. :) 

Yeah, I think for now that makes sense but in the long run, I could see a use case for piping the file paths into imageoptim or something. That's for another day then though. :)

**Comment by stefanjudis** [comment]
Sweet! Thanks a bunch!

**Pull request merged**